test/plugins/validation/2and3/parameters-ibm.js

```diff
@@ -585,5 +585,197 @@ describe('validation plugin - semantic - parameters-ibm', () => {
         'Parameter objects must have a `description` field.'
       );
     });
+
+    it('should return an error for invalid start definitions', () => {
+      const spec = {
+        paths: {
+          '/pets': {
+            get: {
+              parameters: [
+                {
+                  start: 12,
+                  schema: {
+                    type: 'string'
+                  },
+                  description: 'good description',
+                  required: ['url', 'start']
+                }
+              ],
```

Comment:
This is not a valid spec.  A parameter must have "name" and "in".  Also, "required" is just a boolean.